locales/en/landing-all.ftl

```diff
@@ -4,6 +4,13 @@
 
 public-nav-name = { -brand-mozilla-monitor }
 
+## Top navbar
```

Comment:
Leave an empty line before and after group-level comments.

Reply:
updated here:
https://github.com/mozilla/blurts-server/pull/4701/commits/7f80c24e71e8bc5cb29e7c7a148fd4021142c68d